app/src/debug/java/net/squanchy/support/debug/DebugActivity.kt

```diff
@@ -134,4 +143,38 @@ class DebugActivity : AppCompatActivity() {
         OnboardingResetter(this).resetOnboarding()
         Snackbar.make(findViewById<View>(R.id.debug_root), "It's daaaawnnnn", Snackbar.LENGTH_SHORT).show()
     }
+
+    private fun freezeTime() {
+        val now = DebugCurrentTime(applicationContext).currentDateTime()
+
```

Comment:
This empty line would be perfect to put in a "show currently frozen time on a label" thing 💃

Reply:
53ae453